pysat/instruments/madrigal_methods.py

```diff
@@ -212,37 +217,40 @@ def download(date_array, inst_code=None, kindat=None, data_path=None, user=None,
     # TODO, implement user and password values in test code
     # specific to each instrument
     if user is None:
-        print ('No user information supplied for download.')
+        print('No user information supplied for download.')
         user = 'pysat_testing'
     if password is None:
-        print ('Please provide email address in password field.')
+        print('Please provide email address in password field.')
         password = 'pysat_testing@not_real_email.org'
 
     try:
-        a = subprocess.check_output(["globalDownload.py", "--verbose", 
+        a = subprocess.check_output(["globalDownload.py", "--verbose",
                                      "--url="+url, '--outputDir='+data_path,
                                      '--user_fullname='+user,
                                      '--user_email='+password,
                                      '--user_affiliation=pysat',
-                                     '--format='+file_format, \
-                            '--startDate='+date_array[0].strftime('%m/%d/%Y'),\
-                            '--endDate='+date_array[-1].strftime('%m/%d/%Y'),
+                                     '--format='+file_format,
+                                     '--startDate=' +
```

Comment:
Need spaces about '+' here and elsewhere.

Reply:
fixed